### ISSUE TITLE
Browser on multiple platforms

### ISSUE THREAD

**Issue by sylvaincombes** [defunctzombie/zuul#125]
Hi, first thank you for your project :-)

I want to test a browser on multiple platforms with saucelabs.

 I tried a syntax like this :

```yaml
browsers:
  - name: chrome
    version: [27,30,33,latest]
    platform: Windows 8.1

  - name: chrome
    version: [27,30,33,latest]
    platform: Linux
```

But with this, the "last who talk wins" so here only the tests for Linux for example fires.

So I tried this syntax :
```yaml
browsers:
  - name: chrome
    version: [27,30,33,latest]
    platform: ["Windows 8.1","Windows 8","Windows 7","Windows XP","OS X 10.6","OS X 10.8","OS X 10.9","OS X 10.10","Linux"]
```

But this doesn't work either and produce this error :

```javascript
/home/travis/build/sylvaincombes/ju-string/node_modules/zuul/lib/flatten_browser.js:21
                return req.platform.toLowerCase() === browser.platform.toLower
                                    ^
TypeError: Object Windows 8.1,Windows 8,Windows 7,Windows XP,OS X 10.6,OS X 10.8,OS X 10.9,OS X 10.10,Linux has no method 'toLowerCase'
    at /home/travis/build/sylvaincombes/ju-string/node_modules/zuul/lib/flatten_browser.js:21:37
    at Array.filter (native)
    at /home/travis/build/sylvaincombes/ju-string/node_modules/zuul/lib/flatten_browser.js:20:27
    at Array.forEach (native)
    at flatten (/home/travis/build/sylvaincombes/ju-string/node_modules/zuul/lib/flatten_browser.js:12:13)
    at /home/travis/build/sylvaincombes/ju-string/node_modules/zuul/bin/zuul:182:19
    at IncomingMessage.<anonymous> (/home/travis/build/sylvaincombes/ju-string/node_modules/zuul/lib/scout_browser.js:35:13)
    at IncomingMessage.g (events.js:180:16)
    at ClientRequest.<anonymous> (/home/travis/build/sylvaincombes/ju-string/node_modules/zuul/lib/scout_browser.js:28:13)
    at ClientRequest.g (events.js:180:16)
    at ClientRequest.emit (events.js:95:17)
    at HTTPParser.parserOnIncomingClient (http.js:1693:21)
    at HTTPParser.parserOnHeadersComplete [as onHeadersComplete] (http.js:122:23)
    at CleartextStream.socketOnData (http.js:1588:20)
```

Which is normal as it tries to call a string method on an Array object.

Am I missing the good way to achieve this ? 
Or maybe it is not possible yet ? 
If it's not possible can you add support for that ?

Thank you

**Comment by defunctzombie** [comment]
Sounds like a bug to me. We should be supporting this. I think supporting the array of platforms sounds good to me. Not sure when I will be able to code this up yet tho.

**Comment by rase-** [comment]
Merged #131
